ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-ref.yaml

```diff
@@ -25,6 +25,9 @@ ref:
   - name: OPENSHIFT_REQUIRED_MEMORY
     default: "96"
     documentation: The amount of memory assigned to the job.
+  - name: DEFAULT_NETWORK_TYPE
```

Comment:
since you're adding this as the default and a parameter, we might need to take out [this line ](https://github.com/openshift/release/blob/96444bb30c9b2e40da5091181f75759e0c8b91f6/ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-commands.sh#L35)

Reply:
@paigerube14  Good cache! Thanks